Lib/test/test_zipfile/test_core.py

```diff
@@ -3447,6 +3447,53 @@ def test_too_short(self):
         self.assertEqual(
             b"zzz", zipfile._Extra.strip(b"zzz", (self.ZIP64_EXTRA,)))
 
+class StoredZipExtFileRandomAccessTest(unittest.TestCase):
+    def test_random_access(self):
+        from _pyio import BytesIO
+        class StatIO(BytesIO):
+            def __init__(self):
+                super().__init__()
+                self.bytes_read = 0
+
+            def read(self, size=-1):
+                bs = super().read(size)
+                self.bytes_read += len(bs)
+                return bs
+
+            def get_bytes_read(self):
+                return self.bytes_read
```

Comment:
Just directly access the attribute. No need for an additional method (in test modules, OOP is not really important)